TicketsDemo.Data/Entities/BookingServiceAggregate/BookingService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TicketsDemo.Data.Entities
+{
+    public class BookingService
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public int BookingAgencyId { get; set; }
+        public BookingAgency HostAgency { get; set; }
```

Comment:
Either match the names HostAgency and  BookingAgencyId in order to map association to it's key
OR
use fluent api to do the matching (something like 

            modelBuilder.Entity<BookingService>().HasRequired(x => x.HostAgency).WithMany(x => x.BookingServices).HasForeignKey(x => x.BookingAgencyId)WillCascadeOnDelete(false);